adapters/repos/db/bm25f_test.go

```diff
@@ -159,6 +159,26 @@ func TestBM25FJourney(t *testing.T) {
 		// Check explainScore
 		require.Contains(t, res[0].Object.Additional["explainScore"], "BM25F")
 	})
+
+	// Check basic search on string field
```

Comment:
Could you explain a bit what you're testing here? I assume that we're searching for the "offtopic" query and only the following element contains it, which has index 7. Is that correct?
`	testData = append(testData, map[string]interface{}{"title": "An unrelated title", "description": "Absolutely nothing to do with the topic", "stringField": "*&^$@#$%^&*()(offtopic!!!!"})`

Reply:
Added comments to help